index.html

```diff
@@ -1,92 +1,211 @@
+      <div class="card1-btn">
+        <button>css</button>
+        <button>html</button>
+        <button>bootstrap</button>
+        <button>Ruby</button>
+      </div>
+      <button id="card1btn">See Project</button>
+    </div>
+    <div class="card">
+      <h1>Profesional Art Printing Data</h1>
+      <p>A daily selection of privately personalized reads; 
+        no accounts or sign-ups required. 
+        has been the industry's standard</p>
+        <div class="card-btn">
+          <button>html</button>
+          <button>bootstrap</button>
+          <button>Ruby</button>
+        </div>
+        <button class="cardbtn2">See Project</button>
+    </div>
+    <div class="card">
+      <h1>Profesional Art Printing Data</h1>
+      <p>A daily selection of privately personalized reads; 
+        no accounts or sign-ups required. 
+        has been the industry's standard</p>
+        <div class="card-btn">
+          <button>html</button>
+          <button>bootstrap</button>
+          <button>Ruby</button>
+        </div>
+        <button class="cardbtn2">See Project</button>
+    </div>
+    <div class="card">
+      <h1>Profesional Art Printing Data</h1>
+      <p>A daily selection of privately personalized reads; 
+        no accounts or sign-ups required. 
+        has been the industry's standard</p>
+        <div class="card-btn">
+          <button>html</button>
+          <button>bootstrap</button>
+          <button>Ruby</button>
+        </div>
+        <button class="cardbtn2">See Project</button>
+    </div>
+    <div class="card">
+      <h1>Profesional Art Printing Data</h1>
+      <p>A daily selection of privately personalized reads; 
+        no accounts or sign-ups required. 
+        has been the industry's standard</p>
+        <div class="card-btn">
+          <button>html</button>
+          <button>bootstrap</button>
+          <button>Ruby</button>
+        </div>
+        <button class="cardbtn2">See Project</button>
+    </div>
+    <div class="card">
+      <h1>Profesional Art Printing Data</h1>
+      <p>A daily selection of privately personalized reads; 
+        no accounts or sign-ups required. 
+        has been the industry's standard</p>
+        <div class="card-btn">
+          <button>html</button>
+          <button>bootstrap</button>
+          <button>Ruby</button>
+        </div>
+        <button class="cardbtn2">See Project</button>
+    </div>
+    <div class="card">
+      <h1>Profesional Art Printing Data</h1>
+      <p>A daily selection of privately personalized reads; 
+        no accounts or sign-ups required. 
+        has been the industry's standard</p>
+        <div class="card-btn">
+          <button>html</button>
+          <button>bootstrap</button>
+          <button>Ruby</button>
+        </div>
+        <button class="cardbtn2">See Project</button>
+    </div>
+  </section>
+  <section class="about">
+    <div class="card-aboutme">
+      <h1>About me</h1>

```

Comment:
- [ ] You are almost there, however, it is important to use the right font from the template in every part of your project:

Template:
![Captura de pantalla 2022-03-18 a la(s) 7 15 28 p  m](https://user-images.githubusercontent.com/83717285/159098924-acfddb90-7771-423e-882f-a46cc26aa176.png)

Your design:
![Captura de pantalla 2022-03-18 a la(s) 7 15 45 p  m](https://user-images.githubusercontent.com/83717285/159098935-fdaee0d1-cabe-4781-890e-e54781756e1d.png)